src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,140 @@
+package duke;
+
+import java.util.ArrayList;
+
+import duke.task.Task;
+
+/**
+ * The TaskList class.
+ */
+public class TaskList {
+    /**
+     * A task list that stores the user's tasks.
+     */
+    private ArrayList<Task> taskList;
```

Comment:
Would naming the collection of task objects to a plural form such as 'tasks' be better?